torch/ao/quantization/fx/backend_config/native.py

```diff
@@ -92,16 +96,222 @@ def _get_default_op_backend_config(op, dtype_configs):
+        # -----------------
+        # conv relu, fused module
+        conv_configs.append({
+            "pattern": convs.relu,
+            "observation_type": observation_type,
+            "dtype_configs": dtype_configs,
+            "root_module": convs.root,
+            "reference_quantized_module_for_root": convs.reference,
+            "qat_module": convs.relu_qat,
+        })
+        # conv relu, qat fused module
+        conv_configs.append({
+            "pattern": convs.relu_qat,
+            "observation_type": observation_type,
+            "dtype_configs": dtype_configs,
+            "root_module": convs.root,
+            "reference_quantized_module_for_root": convs.reference,
+        })
+        # conv relu, conv module + relu module
+        conv_configs.append({
+            "pattern": (torch.nn.ReLU, convs.root),
+            "observation_type": observation_type,
+            "dtype_configs": dtype_configs,
+            "fuser_method": reverse_sequential_wrapper2(convs.relu),
+        })
+        # conv relu, conv module + functional relu
+        conv_configs.append({
+            "pattern": (F.relu, convs.root),
+            "observation_type": observation_type,
+            "dtype_configs": dtype_configs,
+            "fuser_method": reverse_sequential_wrapper2(convs.relu),
+        })
+        # conv relu, functional conv + relu module
+        conv_configs.append({
+            "pattern": (torch.nn.ReLU, convs.func),
+            "observation_type": observation_type,
+            "dtype_configs": dtype_configs,
+        })
+        # conv relu, functional conv + functional relu
+        conv_configs.append({
+            "pattern": (F.relu, convs.func),
+            "observation_type": observation_type,
+            "dtype_configs": dtype_configs,
+        })
+
+        # (3) Conv + batchnorm (+ relu)
+        # -------------------------------
+        # conv bn, qat fused module
+        conv_configs.append({
+            "pattern": convs.bn_qat,
+            "observation_type": observation_type,
+            "dtype_configs": dtype_configs,
+            "root_module": convs.root,
+            "reference_quantized_module_for_root": convs.reference,
+        })
+        # conv bn relu, qat fused module
+        conv_configs.append({
+            "pattern": convs.bn_relu_qat,
+            "observation_type": observation_type,
+            "dtype_configs": dtype_configs,
+            "root_module": convs.root,
+            "reference_quantized_module_for_root": convs.reference,
+        })
+    return conv_configs
+
+def get_native_backend_config_dict():
+    """ Get backend_config_dict for PyTorch Native backend (fbgemm/qnnpack). """
     return {
         # optional
         "name": "native",
         "configs": [
-            _LINEAR_MODULE_CONFIG,
             *_DEFAULT_OP_INT8_CONFIGS,
             *_DEFAULT_OP_FP16_CONFIGS,
             *_DEFAULT_OP_INT8_OR_FP16_CONFIGS,
+            *_get_linear_configs(),
```

Comment:
I think we also support fp16 linear quantization, how did we support them here in the config?

Reply:
https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fx/_lower_to_native_backend.py#L286

Reply:
(As discussed offline, these will be covered in a future PR)